.devops/code-review-pipelines.yml

```
@@ -12,7 +12,7 @@ resources:
     - repository: pagopaEcommerceLocal
       type: github
       name: pagopa/pagopa-ecommerce-local
-      ref: main
+      ref: CHK-4392-ecommerce-commons-gh-package
```

```suggestion
      ref: main
```
TODO change after https://github.com/pagopa/pagopa-ecommerce-local/pull/233 merge